net/DevExtreme.AspNet.Data/DataSourceLoadOptionsBase.cs

```diff
@@ -7,15 +7,54 @@
 namespace DevExtreme.AspNet.Data {
 
     public abstract class DataSourceLoadOptionsBase {
-        public bool RequireTotalCount { get; set; }
-        public bool IsCountQuery { get; set; }
-        public int Skip { get; set; }
-        public int Take { get; set; }
-        public SortingInfo[] Sort { get; set; }
-        public GroupingInfo[] Group { get; set; }
-        public IList Filter { get; set; }
-        public SummaryInfo[] TotalSummary { get; set; }
-        public SummaryInfo[] GroupSummary { get; set; }
+        public bool RequireTotalCount;
```

Comment:
It is the `DataSourceLoadOptionsBase` class that is the only thing to be extended to meet platform specifics, right?